### PR TITLE
Add azure-pipelines-e2e.yml for Integration tests run

### PR DESCRIPTION
**Pull request by ankitkumarr**

### CODE REVIEW

**Comment by ankitkumarr** [comment]
@pragnagopa, I re-ran the build and it passed. For whatever reason it didn't link here

File: .ci/e2e/setup-e2e.ps1 [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Write-Host "Downloading Functions Core Tools...."
+Invoke-RestMethod -Uri 'https://functionsclibuilds.blob.core.windows.net/builds/2/latest/version.txt' -OutFile version.txt
+Write-Host "Using Functions Core Tools version: $(Get-Content -Raw version.txt)"
+Remove-Item version.txt
+
+$currDir = Get-Location
+$output = "$currDir\Azure.Functions.Cli"
+
+
+if (-not (Test-Path env:CORE_TOOLS_URL))
```
Comment (Francisco-Gamino) [inline .ci/e2e/setup-e2e.ps1]: This should be
``` 
if (-not $env:CORE_TOOLS_URL)
{
}
```

File: .ci/e2e/setup-e2e.ps1 [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+$currDir = Get-Location
+$output = "$currDir\Azure.Functions.Cli"
+
+
+if (-not (Test-Path env:CORE_TOOLS_URL)) 
+{ 
+    $env:CORE_TOOLS_URL = "https://functionsclibuilds.blob.core.windows.net/builds/2/latest/Azure.Functions.Cli.win-x86.zip"
+}
+
+if (Test-Path env:CORE_TOOLS_EXE_PATH)
```
Comment (Francisco-Gamino) [inline .ci/e2e/setup-e2e.ps1]: This should be
``` 
if (-not $env:CORE_TOOLS_EXE_PATH)
{
}
```

File: .ci/e2e/setup-e2e.ps1 (hunk 1 above)
Comment (ankitkumarr) [reply .ci/e2e/setup-e2e.ps1]: We discussed offline, it's ok for now because it's `env:CORE_TOOLS_URL` vs `$env:CORE_TOOLS_URL`.

**Pull request merged**